src/Microsoft.Health.Fhir.Liquid.Converter.Tool/TemplateManagementLogicHandler.cs

```diff
@@ -6,115 +6,80 @@
 
                 OCIFileManager fileManager = new OCIFileManager(options.ImageReference, options.OutputTemplateFolder);
                 if (await fileManager.PullOCIImageAsync())
                 {
                     fileManager.UnpackOCIImage();
-                    Logger.LogInformation($"Succeed to pull templates to {options.OutputTemplateFolder} folder");
+                    Console.WriteLine($"Succeed to pull templates to {options.OutputTemplateFolder} folder");
+                    return 0;
                 }
                 else
                 {
-                    ExceptionHandling(options.ErrorJsonFile, new TemplateManagementException("Fail to pull templates."), "Process Exits: ");
+                    Console.Error.WriteLine($"Fail to pull templates.");
                 }
             }
             catch (Exception ex)
             {
-                ExceptionHandling(options.ErrorJsonFile, ex, "Process Exits: Fail to pull templates. ");
+                Console.Error.WriteLine($"Fail to pull templates. {ex.Message} ");
             }
+
+            return -1;
         }
 
-        internal static async Task PushAsync(PushTemplateOptions options)
+        internal static async Task<int> PushAsync(PushTemplateOptions options)
         {
             try
             {
                 if (!Directory.Exists(options.InputTemplateFolder))
                 {
-                    ExceptionHandling(options.ErrorJsonFile, new Exception($"Input folder {options.InputTemplateFolder} not exist."), "Process Exits: ");
-                    return;
+                    Console.Error.WriteLine($"Process Exits: Input folder {options.InputTemplateFolder} not exist.");
+                    return -1;
                 }
 
                 if (Directory.GetFileSystemEntries(options.InputTemplateFolder).Length == 0)
                 {
-                    ExceptionHandling(options.ErrorJsonFile, new Exception($"Input folder {options.InputTemplateFolder} is empty."), "Process Exits: ");
-                    return;
+                    Console.Error.WriteLine($"Process Exits: Input folder {options.InputTemplateFolder} is empty.");
+                    return -1;
                 }
 
                 OCIFileManager fileManager = new OCIFileManager(options.ImageReference, options.InputTemplateFolder);
                 fileManager.PackOCIImage(options.BuildNewBaseLayer);
                 if (await fileManager.PushOCIImageAsync())
                 {
-                    Logger.LogInformation($"Succeed to push new templates to {options.ImageReference}");
+                    Console.WriteLine($"Succeed to push new templates to {options.ImageReference}");
+                    return 0;
                 }
                 else
                 {
-                    ExceptionHandling(options.ErrorJsonFile, new Exception("Fail to push templates."), "Process Exits: ");
+                    Console.Error.WriteLine("Fail to push templates.");
```

Comment:
Failed to push templates? @BoyaWu10  for grammar help

Reply:
Both of them looks good to me. It seems FHIR Server uses "Failed". I think we can keep consistency here and use "Failed" as well.